### ISSUE TITLE
Model Creation

### ISSUE THREAD

**Issue by Diegomoro415** [Diegomoro415/mildew_detection#17]
- Define a function to create model with Convolutional Neural Network (CNN) architecture.
- Display the summary of the created model
- Implement early stopping to monitor validation loss during training.
